examples/config/owner-onboarding-server.yml

```diff
@@ -13,27 +13,7 @@ owner_addresses:
     - dns_name: fdo.example.com
 report_to_rendezvous_endpoint_enabled: false
 bind: 0.0.0.0:8081
-service_info:
-  sshkey_user: admin
-  sshkey_key: "testkey"
-  files:
-  - path: /device/etc/hosts
-    permissions: 644
-    source_path: /local/etc/hosts
-  - path: /device/etc/resolv.conf
-    source_path: /local/etc/resolv.conf
-  commands:
-  - command: ls
-    args:
-    - /etc/hosts
-    return_stdout: true
-    return_stderr: true
-  - command: ls
-    args:
-    - /etc/doesnotexist/test
-    may_fail: true
-    return_stdout: true
-    return_stderr: true
-  - command: touch
-    args:
-    - /tmp/command-testfile
\ No newline at end of file
+service_info_api_url: "http://localhost:8089/device_info"
+service_info_api_authentication:
+  BearerToken:
+    token: TestAuthToken
```

Comment:
this example should just be:
```toml
+service_info_api_url: "file:///path/to/service_info.json"
+service_info_api_authentication: None
```
as this is what we ship in RPMs too and since we don't ship a service info server we need this

Reply:
ok file:// doesn't work also